simra/simra.go

```diff
@@ -170,7 +170,7 @@ func (sim *simra) RemoveTouchListener(listener peer.TouchListener) {
 // AddCollisionListener add a callback function that is called on
 // collision is detected between c1 and c2.
 func (sim *simra) AddCollisionListener(c1, c2 Collider, listener CollisionListener) {
-	// TODO: exclusive controll
+	// TODO: exclusiveee controll
```

Comment:
__[Misspell]__

"controll" is a misspelling of "control"


<!-- Comment by [SideCI](https://sideci.com) -->